include/dlpack/dlpack.h

```diff
@@ -75,6 +75,8 @@ typedef enum {
    *
    */
   kDLOneAPI = 14,
+  /*! \brief Qualcomm Hexagon DSP */
+  kDLHexagon = 15,
```

Comment:
```suggestion
  kDLHexagon = 16,
```
Quickly proposing the bump to 16 while I remember. In TVM, `15` is used via extension as kDLWebGPU.

Reply:
If there is such a thing as kDLWebGPU it needs to be put here. Without someone knowledgeable with what's out there in the wild to review it, next time the header is updated this spot could be accidentally taken.

Reply:
Good point, @csullivan can you send another PR to reserve WebGPU? Given WebGPU is the nextgen web standard it makes sense to reserve a flag

Reply:
Certainly, #88 is opened.